redis-utils-lib/project.clj

```diff
@@ -0,0 +1,69 @@
+(def redis-distribution
+  "The version of redis to use and hash value of executable. See:
+  https://github.com/antirez/redis-hashes/blob/master/README"
+  {:version "3.2.10"
+   :hash "411c604a716104f7f5a326abfad32de9cea10f15f987bec45cf86f315e9e63a0"})
+
+(def redis-install-path
+  "The directory within this library where redis is to be installed"
+  "redis")
+
```

Comment:
Should these be `defconfig`s instead?

Reply:
defconfig is a macro in cmr common, I wouldn't want to use that in a `project.clj`